AmplifyPlugins/Predictions/AWSPredictionsPlugin/AWSPredictionsPlugin+Reset.swift

```diff
@@ -18,7 +18,6 @@ extension AWSPredictionsPlugin {
         }
 
         if authService != nil {
-            authService.reset()
```

Comment:
Good find. It's probably a good idea to do a quick visual audit of plugins to make sure they're only invoking `reset` on members they own, as opposed to any property that happens to be attached to them.